source/opt/spread_volatile_semantics.cpp

```diff
@@ -75,6 +75,7 @@ bool HasOnlyEntryPointsAsFunctions(IRContext* context, Module* module) {
         entry_point.GetSingleWordInOperand(kOpEntryPointInOperandEntryPoint));
   }
   for (auto& function : *module) {
+    if (function.control_mask() & SpvFunctionControlDontInlineMask) continue;
```

Comment:
This pass should not care about the don't inline mask.  The pass was written assuming everything was inlined to make it simpler.  If you want it to be able to handle the case where a function was not inlined, you have to update the pass.  Avoiding the warning will not make the pass work correctly.

Reply:
The exact case is we have a lot of "empty" dontinline functions in hlsl which are need to be reserved in spirv. and later patched in the llpc/llvm. 

So in our situation, we just need to ignore non-inlined function and the pass does not break the output spirv.

Are we going to propagate the "spread volatile pass" on to non-inline function?